services/director/src/simcore_service_director/config.py

```diff
@@ -125,4 +125,8 @@
     "TRACING_ZIPKIN_ENDPOINT", "http://jaeger:9411"
 )
 
+# resources
+CPU_RESOURCE_LIMIT_KEY = "SIMCORE_NANO_CPUS_LIMIT"
+MEM_RESOURCE_LIMIT_KEY = "SIMCORE_MEMORY_BYTES_LIMIT"
+

```

Comment:
do not forget to remove these

Reply:
I would if I could but I can't so I shan't.